forum/investors/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from startups.models import Location
+import uuid
+
+
+
+class Investor(models.Model):
+    """
+    Model representing an investor.
+
+    Attributes:
+        user (ForeignKey): The user associated with the investor.
+        company_name (str): The name of the investor's company.
+        available_funds (decimal): Available funds for investment.
+        experience_years (int): The years of experience of the investor.
+        location (ForeignKey): The location associated with the investor.
+        company_logo (file): Optional company logo.
+        description (str): Description of the investor's company.
+        website (str): The website of the company.
+        created_at (datetime): Date when the investor was created.
+    """
+    investor_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    user = models.ForeignKey('User', on_delete=models.CASCADE, related_name='investors', db_index=True)
+    company_name = models.CharField(max_length=150, unique=True, db_index=True)
+    available_funds = models.DecimalField(max_digits=15, decimal_places=2)
```

Comment:
Consider adding validation to ensure non-negative values since available funds cannot be negative.
```
available_funds = models.DecimalField(max_digits=15, decimal_places=2, validators=[MinValueValidator(0)])

```